plugins/builtin/apm/nomad/plugin/metrics.go

```diff
@@ -27,7 +29,7 @@ const (
 )
 
 // Query satisfies the Query function on the apm.APM interface.
-func (a *APMPlugin) Query(q string) (float64, error) {
+func (a *APMPlugin) Query(q string, _ sdk.TimeRange) (sdk.TimestampedMetrics, error) {
```

Comment:
because nomad doesn't support historical data, right?

Reply:
Yup. I will highlight that in a comment.